React/src/components/CloudAdmin/CloudAdmin.tsx

```diff
@@ -0,0 +1,188 @@
+                        }}
+                    >
+                        Filtruj
+                    </Button>
+                </Grid>
+                <Grid item xs={10}>
+                    <TableContainer component={Paper} sx={{ backgroundColor: '#1A1C26' }}>
+                        <Table aria-label="simple table">
+                            <TableHead>
+                                <TableRow>
+                                    <StyledTableCell colSpan={5}>
+                                        <Typography variant="h5" fontWeight={"bold"}>Lista firm</Typography>
+                                    </StyledTableCell>
+                                </TableRow>
+                                <TableRow>
+                                    <StyledTableCellHeader>Id</StyledTableCellHeader>
+                                    <StyledTableCellHeader align="left">Nazwa firmy</StyledTableCellHeader>
+                                    <StyledTableCellHeader align="left">Data utworzenia</StyledTableCellHeader>
+                                    <StyledTableCellHeader align="left">Status</StyledTableCellHeader>
+                                    <StyledTableCellHeader align="left">Akcje</StyledTableCellHeader>
+                                </TableRow>
+                            </TableHead>
+                            <TableBody>
+                                {
+                                    data && data.map(company => (
+                                        <TableRow>
+                                            <StyledTableCell>{company.id}</StyledTableCell>
+                                            <StyledTableCell align="left">{company.name}</StyledTableCell>
+                                            <StyledTableCell align="left">{company.date}</StyledTableCell>
+                                            <StyledTableCell align="left">{company.status}</StyledTableCell>
+                                            <StyledTableCell align="left" style={{ textAlign: "center" }}>
+                                                <Link to='/company-list/edit/{company.id}'>
+                                                    <ToolElementButton>Edytuj</ToolElementButton>
+                                                </Link>
+                                                <Link to='/company-list/block/{company.id}'>
+                                                    <ToolElementButton>Blokuj</ToolElementButton>
+                                                </Link>
+                                            </StyledTableCell>
+                                        </TableRow>
+                                    ))
+                                }
+                            </TableBody>
+                        </Table>
+                    </TableContainer>
+                </Grid>
+            </Grid>
+        </Box>
+    );
+
+    return content;
+}
+
+export default CloudAdmin;
```

Comment:
It should be done like this:
https://github.com/s20146/PRO/tree/master/React/src/pages/orders